apps/fm/js/fm.js

```diff
@@ -870,47 +872,29 @@ function init() {
     window.dispatchEvent(new CustomEvent('moz-app-loaded'));
   });
 
-  //
-  // If the system app is opening an attention screen (because
-  // of an incoming call or an alarm, e.g.) and if we are
-  // currently playing the radio then we need to stop the radio
-  // before the ringer or alarm starts sounding. See bugs 995540
-  // and 1006200.
-  //
-  // XXX We're abusing the settings API here to allow the system app
-  // to broadcast a message to any certified apps that care. There
-  // ought to be a better way, but this is a quick and easy way to
-  // fix a last-minute release blocker.
-  //
-  navigator.mozSettings.addObserver(
-    'private.broadcast.attention_screen_opening',
-    function(event) {
-      // An attention screen is in the process of opening. Save the
-      // current state of the radio and disable.
-      if (event.settingValue) {
-        window._previousFMRadioState = mozFMRadio.enabled;
-        window._previousEnablingState = enabling;
-        window._previousSpeakerForcedState = speakerManager.speakerforced;
-        mozFMRadio.disable();
-      }
+  // Compete for the content audio channel
```

Comment:
Please expand this comment and explain when the audio channel will be interrupted.